tests/test_repo_interactions.py

```diff
@@ -9,6 +11,15 @@
 from chartpress import PRERELEASE_PREFIX, yaml
 
 
+@contextlib.contextmanager
```

Comment:
We've got several pytest fixtures in https://github.com/jupyterhub/chartpress/blob/main/tests/conftest.py which behave as context managers, would you mind following that pattern?

Reply:
Sure thing

Reply:
Parameterizing removed the need for this, so I just removed it